src/components/ClickableStyle/__snapshots__/ClickableStyle.test.ts.snap

```diff
@@ -0,0 +1,177 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Maintaining `ClickableStyles` for now, as the main reason it gets exported is to facilitate some behavior in Remix, and some possible edge cases that aren't buttons or links, but should be clickable.